README.md

```diff
@@ -59,7 +59,10 @@ The **Run MATLAB Tests** action lets you customize your test run using optional
 Input                     | Description
 ------------------------- | ---------------
 `source-folder`            | (Optional) Location of the folder containing source code, relative to the project root folder. The specified folder and its subfolders are added to the top of the MATLAB search path. If you specify `source-folder` and then generate a coverage report, MATLAB uses only the source code in the specified folder and its subfolders to generate the report. You can specify multiple folders using a colon-separated or semicolon-separated list.<br/>**Example:** `source`<br/>**Example:** `source/folderA; source/folderB`
+`strict`                    | (Optional) Whether to apply strict checks when running the tests. For example, the action generates a qualification failure if a test issues a warning.
 `use-parallel`              | (Optional) Whether to run tests in parallel on a self-hosted runner, specified as `false` or `true`. By default, the value is `false` and tests run in serial. If the test runner configuration is suited for parallelization, you can specify a value of `true` to run tests in parallel. This input requires a Parallel Computing Toolbox™ license and is supported only on self-hosted runners.
+`output-detail`             | (Optional) Display level for event details produced by the test run, specified as `None`, `Terse`, `Concise`, `Detailed`, or `Verbose`.
+`logging-level`             | (Optional) Maximum verbosity level for logged diagnostics included for the test run, specified as `None`, `Terse`, `Concise`, `Detailed`, or `Verbose`.
```

Comment:
Should we keep these new features listed here or move it to the bottom (including `use-parallel`)?

Reply:
Hi Kapil. I will take care of README.md after you close your pull request. This will be in a separate doc pull request.

Reply:
Okay, thanks Houman!

Reply:
Sure, Kapil. Let's keep the code and doc updates separate as before. Thanks.